.github/workflows/frontend-e2e-test.yml

```diff
@@ -37,31 +37,54 @@ jobs:
       - name: Setup golang environment
         uses: actions/setup-go@v2.1.3
         with:
-          go-version: "1.13"
+          go-version: '1.13'
```

Comment:
ditto

Reply:
Got it.

Reply:
The rest of the formatting has been changed. The official documentation for this place uses single quotes, so let's leave it as it is, probably having changed the formatting wrong before.

![image](https://user-images.githubusercontent.com/31329157/113300750-d4a18300-9330-11eb-8438-244fbdce0800.png)